api/swagger.yml

```
@@ -298,12 +298,40 @@ paths:
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ReportTemplateOption"
+        403:
+          $ref: "#/components/responses/Forbidden"
+        500:
+          $ref: "#/components/responses/Internal"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)